source/reference/limitations.txt

```diff
@@ -32,16 +32,8 @@ General Limitations
   6.2. Only major MongoDB releases such as 6.0 or 7.0. are supported.
   For more information on MongoDB versioning, see
   :ref:`release-version-numbers`.
-- The source and destination clusters must have the same major and minor
```

Comment:
Could we revert the first bullet in the general limitations section to say `The minimum supported server version is MongoDB 6.0.5.` ?

Reply:
Fixed